test/utils/fieldUtils.js

```diff
@@ -118,6 +118,18 @@ class FieldDataTypes {
     return dateField;
   }
 
+  /**
+   * Enter the valid document file name to be uploaded
+   * @returns CCDStringField Object
+   */
+  async enterIntoDocumentField(value, id){
+    let css = await FIELDS.DOCUMENT.cssTag;
+    let docField = new CCDDocumentField(css, id);
+    await docField.uploadFile('C:\\test\\framework-structure.png');
+    await browser.sleep(3000);
```

Comment:
we should try to avoid hardcoded sleeps where possible, does the test fail without the sleep?

Reply:
Removed.